test/Options/SimpleOption.js

```diff
@@ -1,21 +1,358 @@
 const {
```

Comment:
Small comment on formatting, i suggest we all agree on and use an auto formatter for all languages and then just aut format before checkin. Not a big issue here, but going forward on a project it's nice if code all looks the same so it is easy to reas without format being noticed.